build/package/linux/rpm/index.sh

```diff
@@ -12,6 +12,10 @@ echo -e "$RPM_KEY_PRIVATE" | gpg --import --batch
 
 cd /packages/rpm
 
+# Clear (if previous build failed)
+rm -rf .repodata || true
+rm -rf repodata.old.* || true
+
```

Comment:
Musime to mazat aj pred prikazom, ak subory zostali, po predchadzajucom neuspesnom builde :man_facepalming: 

![image](https://user-images.githubusercontent.com/19371734/154963198-284e73a8-c46c-4830-ad0f-17918ebe1b61.png)

Ak sa nepodari vytvorit zamok, tak prikaz skonci s kodom `0`, preto sa nerobilo ani restry.

Reply:
https://github.com/keboola/keboola-as-code/runs/5274503673?check_suite_focus=true

Ale zvysok vyzera byt uz ok, tak snad posledny problem.